Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -727,7 +731,7 @@
 				CODE_SIGN_ENTITLEMENTS = "Basic-Car-Maintenance/Basic_Car_Maintenance.entitlements";
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_ASSET_PATHS = "\"Basic-Car-Maintenance/Preview Content\"";
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = MW75NQ7TG5;
```

Comment:
```suggestion
				DEVELOPMENT_TEAM ="";
```

Reply:
Click these "commit suggestion" buttons after you make your changes, it's someone with the project that I messed up and will be fixing today